src/main/java/com/cmsc495/ticketsystem/controller/LoginController.java

```diff
@@ -0,0 +1,20 @@
+/*Controls redirects for login/logout*/
+
+package com.cmsc495.ticketsystem.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class LoginController {
+    @GetMapping("/login")
+    public String showLoginPage() {
+        return "customLogin";
```

Comment:
login.html should be fine for the naming.

In the future, html convention says custom-login.html is proper

Reply:
done